src/modules/nodejs.rs

```diff
@@ -59,12 +73,45 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     Some(module)
 }
 
+fn get_engines_version(base_dir: &PathBuf) -> Option<String> {
+    if let Ok(json_str) = utils::read_file(base_dir.join("package.json")) {
+        let package_json: json::Value = json::from_str(&json_str).ok()?;
+        let raw_version = package_json.get("engines")?.get("node")?.as_str()?;
+        Some(raw_version.to_string())
+    } else {
+        None
+    }
```

Comment:
Not sure if it matters but you could simplify this a little bit to:
```suggestion
    let json_str = utils::read_file(base_dir.join("package.json")).ok()?;
    let package_json: json::Value = json::from_str(&json_str).ok()?;
    let raw_version = package_json.get("engines")?.get("node")?.as_str()?;
    Some(raw_version.to_string())
```